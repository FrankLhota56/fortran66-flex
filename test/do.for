C
C DO loop test
C
      Do99I = 1,5
      Do99I = 1.5
99    Continue
      End
