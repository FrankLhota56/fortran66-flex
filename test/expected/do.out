Line     4: DO
Line     4: INT_LIT(99)
Line     4: ID(i)
Line     4: '='
Line     4: INT_LIT(1)
Line     4: ','
Line     4: INT_LIT(5)
Line     5: EOS
Line     5: ID(do99i)
Line     5: '='
Line     5: REAL_LIT(1.500000)
Line     6: EOS
Line     6: INT_LIT(99)
Line     6: CONTINUE
Line     7: EOS
Line     7: END
